ironic-deployment/ironic/ironic.yaml

```diff
@@ -17,6 +17,7 @@ spec:
         name: ${NAMEPREFIX}-ironic
     spec:
       hostNetwork: true
+      minReadySeconds: 10
```

Comment:
`minReadySeconds` should be under the topmost `spec` field.

Reply:
Thanks, fixed!